test_scripts/Smoke/API/039_PerfomInteraction_Non_Media_PositiveCase_SUCCESS.lua

```diff
@@ -127,11 +127,11 @@ local function SendOnSystemContext(self, ctx)
     { appID = commonSmoke.getHMIAppId(), systemContext = ctx })
 end
 
---! @setExChoiseSet: ChoiceSet structure for UI.PerformInteraction request
+--! @setExChoiceSet: ChoiceSet structure for UI.PerformInteraction request
```

Comment:
@GetmanetsIrina Please update function name of function `setChoiseSet()` too. (it has the same missprint)